content/swiftui/concepts/viewmodifier/terms/overlay/overlay.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: '.overlay()'
+Description: 'Layers the specified views in front of another view.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Modifiers'
+  - 'Xcode'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.overlay()`** layers the specified [views](https://www.codecademy.com/resources/docs/swiftui/views) in front of another view.
+
+## Syntax
+
+```pseudo
+Text("This text will have views layered over top of it.")
+	.overlay(alignment) {
+		// Subviews here
+}
+```
+
+The `.overlay()` modifier method takes two parameters to create a [`ZStack`](https://www.codecademy.com/resources/docs/swiftui/views/terms/zstack) to group the foreground views:
+
+- `alignment` specifies where to position the foreground views.
+- `content` is used to declare the views to add to the foreground, stacked in the order listed. The last view appears at the front of the stack.
+
+Using the `.overlay()` modifier method without any specified parameter results in the default behavior of the modifier: the foreground views will be centered on the view.
+
+## Example
+
+The following example uses the `.overlay` modifier to lay a star and a [`Circle`](https://www.codecademy.com/resources/docs/swiftui/views/terms/circle) over a over a `Color` view.:
+
+```swift
+Color.blue
+	.frame(width: 200, height: 200)
```

Comment:
For this example can we also change the tabbed indentation to two spaces instead? So for every tab replace with two spaces instead. For example, if there is one tab then change to two spaces. If there are two tabs then four spaces would replace them. Three tabs = six spaces and so on.